### ISSUE TITLE
Toolbar items are hidden when editor's container is hidden on creation

### ISSUE THREAD

**Issue by cabralRodrigo** [nhn/tui.editor#1674]
## Describe the bug
When the editor is created on a hidden div, the toolbar items are hidden.

## To Reproduce
1. Create the editor on a hidden div
2. Show the div
3. All toolbar items are hidden inside de 'more' item.

## Expected behavior
When the editor is shown, I excepted the toolbar items were visible as normal:
![image](https://user-images.githubusercontent.com/5928832/125352479-b38cfd00-e337-11eb-85a8-074e693e6fa0.png)

## Screenshots
After the editor is shown, all toolbar items are hidden:
![image](https://user-images.githubusercontent.com/5928832/125352420-9ce6a600-e337-11eb-99e8-c72e77803a03.png)

## Desktop:
 - OS: Windows 10 64bit
 - Browser: Chrome 91.0.4472.124
 - Version: 3.0.1

## Additional context
Codepen with demo: https://codepen.io/cabralRodrigo/pen/abWBGQR

**Comment by js87zz** [comment]
@cabralRodrigo 
It is recommended to use the Editor API instead of manipulating the style of container element. 
It works well in the case below~!
```html
<h2>Shown</h2>
<div id="editor-shown"></div>
<br>
<h2>Hidden <button id="show-hidden">Show hidden</button></h2>
<!-- remove existing style --> 
<div id="editor-hidden"></div>
```
```js
document.addEventListener('DOMContentLoaded', () => {
  new toastui.Editor({ el: containerShown });
  editor = new toastui.Editor({ el: containerHidden });
  editor.hide()
});

button.onclick = () => {
  editor.show();
}
```

**Comment by cabralRodrigo** [comment]
Thank you for the reply.
The example that I provided was not the real case scenario that I encountered.
I'm using the editor inside a bootstrap collapse. 
So the div container it's not what is hidden, but a parent element.

The example shows a simpler scenario when the bug happen too.

**Comment by js87zz** [comment]
@cabralRodrigo 
OK! 
I'll look into the details later. There is a backlog of other priorities at the moment.🙏

**Comment by cabralRodrigo** [comment]
Ok, thank you for the attention :)

**Comment by davidgeary** [comment]
The problem appears to be in [toolbar.ts](https://github.com/nhn/tui.editor/blob/3983510e871bf4cfdd98131a45e447d8cad1fd86/apps/editor/src/ui/components/toolbar/toolbar.ts#L194), where it's checking the client width in order to calculate whether the toolbar fits.  So if the editor or its parent is not visible, this is likely to be zero, causing the toolbar to be hidden.

This may actually be quite complicated (or maybe even impossible) to fix.  In my use case, I'm using Angular to display the editor in a modal dialog.  This means that, on instantiation, the editor is not actually part of the page's DOM - the dialog is added/removed from the DOM as its visibility changes.  So I'm not sure how the editor would detect that.

Resizing the window works because the `resize` event is wired up to a [call to setState](https://github.com/nhn/tui.editor/blob/3983510e871bf4cfdd98131a45e447d8cad1fd86/apps/editor/src/ui/components/toolbar/toolbar.ts#L124), which recalculates whether the toolbar will fit the new window size.  So as a workaround, calling the following when the editor/parent's visibility is changed will work:

``` javascript
window.dispatchEvent(new Event('resize'));
```

@js87zz I can't see anything on the `Editor` or `EditorCore` classes that can be called to trigger a refresh of the toolbar, but a method that simply called `toolbar.handleResize()` on each toolbar would be a good solution.

**Comment by js87zz** [comment]
@cabralRodrigo @davidgeary 
The bug has been fixed in `v3.0.3`.

**Comment by cabralRodrigo** [comment]
Thank you very much @js87zz !
